src/__tests__/__snapshots__/index.spec.tsx.snap

```diff
@@ -1176,24 +1163,11 @@ exports[`HTML Output given write mode, should populate proper nodes 1`] = `
   </div>
   <div>
     <div>
-      <div style=\\"margin-left: 20px;\\">
-        <div>
-          <div>id</div>
-          <span class=\\"text-green-7 dark:text-green-5\\">string</span>
-        </div>
-        <span class=\\"text-darken-7 dark:text-lighten-6\\">read-only</span>
-        <div></div>
-      </div>
-    </div>
-  </div>
-  <div>
-    <div>
-      <div style=\\"margin-left: 20px;\\">
+      <div style=\\"margin-left: 20px\\">
```

Comment:
is this change expected?

Reply:
Yeah. We had a test that actually didn't pass `viewMode`.
See https://github.com/stoplightio/json-schema-viewer/pull/95/files#diff-c9e3b92b4cfc316d4d981b2e7a03b7ba1648da9bcbcb0ae226133d38027890ebR130